terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_version = "~> 1.3.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "secrets" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.191"
```

Comment:
for extensions-web , we were asked to change the "secrets" source line to a newer version
  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.192"

and for the "ecs-service" source - 
  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/ecs-service?ref=1.0.192"